py-polars/tests/unit/test_selectors.py

```diff
@@ -338,6 +338,10 @@ def test_selector_repr() -> None:
         repr(cs.integer() & cs.matches("z"))
         == "(cs.integer() & cs.matches(pattern='z'))"
     )
+    assert (
+        cs.temporal() | cs.by_dtype(pl.Utf8) & cs.string(False)
+        == "(cs.temporal() | (cs.by_dtype(dtypes=[Utf8]) & cs.string(include_categorical=False)))"
```

Comment:
Not sure if  `include_categorical` should take into `cs.string`'s repr. If yes, perhaps we should pass it to `_selector_proxy_`'s `__init__`?

Reply:
We already do - that's what `parameters` is for, in order to show what the higher-level selector was instantiated with.